lib/autocomplete-manager.coffee

```diff
@@ -190,8 +191,16 @@ class AutocompleteManager
     return unless suggestions.length
 
     unless @overlayDecoration?
-      marker = @editor.getLastCursor()?.getMarker()
-      @overlayDecoration = @editor?.decorateMarker(marker, { type: 'overlay', item: this })
+      if atom.config.get('autocomplete-plus.selectionsListFollows') == 'Cursor'
+        @currentWordMarker = @editor.getLastCursor()?.getMarker()
+      else
+        cursor = @editor.getLastCursor()
+        position = cursor?.getBeginningOfCurrentWordBufferPosition()
+        @currentWordMarker = @editor.markBufferPosition(position)
+
+      @compositeDisposable.add @currentWordMarker
```

Comment:
You will only want to add the current word marker, to the content disposable, if you are creating it in your else case.

This is probably why the tests are failing since they current word marker shouldn't be disposed by our view if it is following the cursor.  

Also we still need a test for this new case.